Topics/Development_Process/clean_coding_styling.md

```diff
@@ -0,0 +1,37 @@
+# Clean coding: coding and comment styling!
+
+
+### Why care about coding and commenting style?
+
+Code isn't good just cause it works. As we become more experienced in software engineering we soon realize that there is so much to consider other than the fact that it works. In fact, functioning code is the bare minimum. For example, most of us have learned that we need to consider time and space efficiency. If we didn't we'd constantly have to wait for the results we want. This can pose a big problem if those results are dependent on other results that are also not time efficient. We also have to think about scalability. We don't want to have to rewrite code just to add one feature. However, what's so important about coding and commenting style? How does that affect users? If I understand it is that not good enough? In short, no.
+
+First, we should consider the event of inconsistent styling. This can mean not using naming conventions, inconsitent documentation/ commenting, and inconsistent spacing. In this case, while you might understand what is going on as you are writing it, future will most likey get confused (especially if you haven't worked with the same language in a while). For example, consistent naming conventions can serve as a hint of the purpose of whatever it is you're naming. For example, specific naming conventions can tell you what type of function you are looking at (for example, a unit test function vs. a use case function), what type of variable (different scopes), etc. Consistent commenting and spacing also add to the readability of your code. Therefore, future you will thank you greatly for giving them hints on how to understand your code. Not just future you but other developers that will have to work with your code.
```

Comment:
'future will most likey' seems to be missing some words, and there is also a typo on the word 'likely'. I believe you are trying to say something like 'future you will most likely get confused'.